### DISCUSSION TITLE
Ble BlueTooth

### DISCUSSION THREAD

**Discussion by hzc-byte** [wh201906/SerialTest#14] [Q&A]
发现低功耗蓝牙连接上了，但是没有数据输出
我自己写的一个连接蓝牙的功能是有数据输出的
不清楚是什么情况

**Comment by wh201906** [comment]
可以描述一下具体的使用情况吗？比如程序是在什么上面运行的？和什么设备建立蓝牙连接？

**Comment by wh201906** [comment]
目前最新版(V0.3.5)对蓝牙BLE的支持会好很多，你可以再试一下
